android/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ add_library(
         src/main/cpp/java-bindings/JFrameProcessorPlugin.cpp
         src/main/cpp/java-bindings/JImageProxy.cpp
         src/main/cpp/java-bindings/JHashMap.cpp
+        ${NODE_MODULES_DIR}/react-native/ReactCommon/jsi/jsi/jsi.cpp
```

Comment:
This should be avoided if possible. Does it really not build if that line is removed? If not, then please make this conditional (`if` + `set`), so it is not compiled in RN versions that support the `JSI_LIB` library below.

Reply:
aka: On RN 0.64 and below, compile `jsi.cpp` here. On RN 0.65 and above, do NOT compile `jsi.cpp` here, but find the `JSI_LIB` below.

Reply:
the reason for that is, that on RN 0.65 and above, `jsi.cpp` is compiled multiple times - aka we have duplicate symbols. This is a problem in C++, since JSI now exists more than once. My `jsi::JSError` for example is not the same as `jsi::JSError` in React Native, therefore the app hard-crashes. So we need to make that conditional. Check out Reanimated codebase on how they do it